param/low_level_motion_0.05.yaml

```diff
@@ -40,3 +40,5 @@ roll_max_rate     : 1.0
 yaw_max           : 3.0
 yaw_min           : -3.0
 yaw_max_rate      : 600.0
+
+update_frequency: 60.0
```

Comment:
Sorry, you're right. I meant its usage in the quadvelocitycontroller. Not something to work with now though.